spanner/google/cloud/spanner_admin_database_v1/gapic/database_admin_client_config.py

```diff
@@ -23,7 +23,7 @@
                     "retry_params_name": "default",
                 },
                 "CreateDatabase": {
-                    "timeout_millis": 30000,
+                    "timeout_millis": 3600000,
```

Comment:
@crwilcox, @snehashah16 Are we actually going to have users wait for an hour before raising en error?

Reply:
Yep. There is an error in the LRO that blocks instead of returning. This is really the safest option at this time. :(